src/components/Modals/ManageAccounts/ManageAccountsModal.tsx

```diff
@@ -112,14 +105,24 @@ export const ManageAccountsModal = () => {
 
   const disableAddChain = walletConnectedChainIdsSorted.length >= availableChainIds.length
 
+  // don't allow users to close the modal until at least one chain is connected
+  const disableClose = walletConnectedChainIdsSorted.length === 0
```

Comment:
V. nice one, confirmed that blocks both close button click as well as clicking outside the modal 🧠